snap/helpers/directorySetup.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+SDATA=''; SCOMMON='';
+if env | grep -q root; then
+  export SDATA=$(echo $SNAP_DATA | sed s!$SNAP_REVISION!current!)
+  export SCOMMON="$SNAP_COMMON"
+  export isRoot=`true`
+else
+  echo "You are not running gitea as root. This is required for the snap package."
+  echo "Please re-run as root."
```

Comment:
Why is it requried to run Gitea as root? (I am not familiar with snap-things)

Reply:
There could be a way to run it as a user of started manually. While working on the gigs snap though, that got messy very fast. This I stepped that out to be revisited at a later point.

Reply:
Sorry, typing on mobile:
if started*
s/gigs/gogs/
Thus I backed that*

Reply:
Hi @tbraeutigam, sorry for bringing this up 2 years after the PR, but do you recall exactly which issue prevented `gitea web` from running correctly without daemon mode?